static/locale/ca/LC_MESSAGES/django.po

```diff
@@ -4188,7 +4188,11 @@ msgstr ""
 msgid "You can only invite other Zephyr mirroring users to private streams."
 msgstr ""
 
-#: zerver/views/streams.py:329
+#: zerver/views/streams.py:336
+msgid "Only administrators can modify other users' subscriptions."
+msgstr ""
+
+#: zerver/views/streams.py:338
```

Comment:
You don't need to and should not modify translation files -- these are automatically generated from Transifex and the rest of the codebase.

Reply:
Fixed, thanks!